fpnge.h

```diff
@@ -39,6 +41,9 @@ inline void FPNGEFillOptions(struct FPNGEOptions *options, int level) {
   case 3:
     options->predictor = 5;
     break;
+  case 5:
+    options->huffman_sample = 23;
+    // fall through
```

Comment:
Maybe `[[fallthrough]];`? (if we're fine with C++17 - I personally am)

Reply:
That'd break C compatibility.

Reply:
Ah I didn't notice this is in the header. Nevermind then :)